resources/public/api/conf/3.0/retrieve_property_bsas.raml

```diff
@@ -2,15 +2,12 @@ is:
   - headers.acceptJson
   - headers.testScenarioHeader
   - errors.formatNino
-  - errors.formatBsasId
-  - errors.formatAdjustedStatus
+  - errors.formatCalculationId
   - errors.ruleNotUkProperty
```

Comment:
This ruleNotUkProperty description does not match the spec. This one refers to the `BSAS ID` where the spec refers to the `calculation ID`